docs/utilities/tags.md

```diff
@@ -11,10 +11,9 @@ For example, to add your own mod's saplings to the Vanilla sapling tag, you woul
 If `replace` is true, then all entries before the json specifying `replace` will be removed.
 See the [Vanilla wiki][tags] for a description of the base syntax.
 
-Forge provides two extensions on the Vanilla syntax:
+Forge extensions on the Vanilla syntax:
```

Comment:
Not really a proper way to define it. This can be removed in favor of just listing it as part of the description.